doc/admin/admin_commands/kdb5_util.rst

```diff
@@ -197,8 +209,8 @@ load
 
 .. _kdb5_util_load:
 
-    **load** [**-b7**\|\ **-ov**\|\ **-r13**] [**-hash**]
-    [**-verbose**] [**-update**] *filename* [*dbname*]
+    **load** [**-b7**\|\ **-ov**\|\ **-r13**\|\ **-r18**] [**-hash**]
```

Comment:
Comment here should be updated or removed: https://github.com/krb5/krb5/blob/master/src/kadmin/dbutil/dump.c#L1497 (also same deal with `-i` and `-c`).

Reply:
I decided to update this comment as there is a similar up-to-date one for dump.